doc/using/QuickStartGuide.rst

```diff
@@ -234,9 +234,9 @@ tips are:
   rules.  Use ``--std=08`` for VHDL-2008 (albeit not fully
   implemented).  All the units must be analyzed with the same standard.
 
-* Use ``--work=NAME`` to analyze files into the ``NAME`` library.
+* Use ``--work=LIB_NAME`` to analyze files into the ``LIB_NAME`` library.
   If you analyze other files from a different directory, give the path
-  of the ``NAME`` library using ``-P/PATH/TO/NAME/DIRECTORY/``.
+  of the ``LIB_NAME`` library using ``-P/PATH/TO/NAME/DIRECTORY/``.
```

Comment:
This is just a matter of style, but I'd use `-P/path/to/name/directory/`. We prefer this format:

- https://ghdl.readthedocs.io/en/latest/getting/GCC.html
- https://ghdl.readthedocs.io/en/latest/getting/LLVM.html
- https://ghdl.readthedocs.io/en/latest/getting/index.html?highlight=path#building-ghdl-from-sources
- https://ghdl.readthedocs.io/en/latest/references/CommandReference.html#gcc-llvm-only-commands

Nonetheless, capitalized paths are still found:

- https://ghdl.readthedocs.io/en/latest/using/QuickStartGuide.html#starting-with-your-design

Hence, once again, it is up to you.